RELEASE.md

```diff
@@ -0,0 +1,3 @@
+Release type: minor
```

Comment:
I think it would way better to move this in the [changelog](https://github.com/getpelican/pelican/blob/master/docs/changelog.rst)

Reply:
Actually, @willthong was performing precisely the steps that contributors are asked to do, in this case the relevant step is the [3rd bullet point in the Contributing docs](https://docs.getpelican.com/en/latest/contribute.html#using-git-and-github). The information Will provided here in the `RELEASE.md` file is used by [AutoPub](https://justinmayer.com/projects/autopub/) in the [GitHub Actions CI workflow](https://github.com/getpelican/pelican/blob/b8bf5950b6a6c684196a6d047d41efbff0f95c85/.github/workflows/main.yml#L96-L136) to increment the version number, update the change-log, publish the GitHub release, and upload the package to PyPI — all of which happens automatically when the PR is merged. 🌟 

In the case of this specific PR, I will probably remove the `RELEASE.md` file before merging because there are a few things I'd like to do issuing a new release. But Will did absolutely the right thing by including it. Well done, Will! 👏

Reply:
ok, sorry :-)